resources/i18n/hi_HI/fdmprinter.def.json.po

```diff
@@ -0,0 +1,8786 @@
+# Cura
+# Copyright (C) 2021 PBR Research
+# This file is distributed under the same license as the Cura package.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Cura 4.13\n"
+"Report-Msgid-Bugs-To: ksonone8@gmail.com\n"
+"POT-Creation-Date: 2021-10-15 11:35+0000\n"
+"PO-Revision-Date: 2021-10-27 1:22+0200\n"
+"Last-Translator: Komal R. Gupta, Kapil H. Sonone <ksonone8@gmail.com>\n"
+"Language-Team: PBR Research <www.github.com/pbr-research>\n"
+"Language: hi_HI\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_settings label"
+msgid "Machine"
+msgstr "मशीन"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_settings description"
+msgid "Machine specific settings"
+msgstr "मशीन विशिष्ट सेटिंग्स"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_name label"
+msgid "Machine Type"
+msgstr "मशीन का प्रकार"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_name description"
+msgid "The name of your 3D printer model."
+msgstr "आपके 3D प्रिंटर मॉडल का नाम।"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_show_variants label"
+msgid "Show Machine Variants"
+msgstr "मशीन के प्रकार दिखाएं"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_show_variants description"
+msgid "Whether to show the different variants of this machine, which are described in separate json files."
+msgstr "क्या इस मशीन के विभिन्न प्रकारों को दिखाना है, जिनका वर्णन अलग-अलग json फाइलों में किया गया है।"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_start_gcode label"
+msgid "Start G-code"
+msgstr "G-code शुरू करें"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_start_gcode description"
+msgid ""
+"G-code commands to be executed at the very start - separated by \n"
+"."
+msgstr ""
+"G-code कमांड को बहुत शुरुआत में निष्पादित किया जाना है.\n - से अलग किया गया"
+"."
```

Comment:
This is weird. The g-code commands need to be separated by `\n`. In the English text this is written as a phrase at the end: `- separated by \n.` But in your text the `\n` is before the `-`, so how could that be right?